tests/basics/TrickAssignments32.py

```diff
@@ -510,7 +510,7 @@ def __int__(self):
             nonlocal a
             del a
 
-            return False
```

Comment:
What warning were you talking about? I tended to purposefully no return booleans, as that must work as well.

Reply:
```
DeprecationWarning: __int__ returned non-int (type bool).  The ability to return an instance of a strict subclass of int is deprecated, and may be removed in a future version of Python.
  c = C()
```
This is the message I get when using both Nuitka and default Python interpreter.
```
Python 3.7.3
```

Reply:
@kayhayen , so, should I change this back or its OK?

Reply:
I meant to say something about this, the test runner ignores warnings. The test code often does bad things, but Nuitka is not about compiling good code, but one that actually exists. You ought to use `./run_all.py filename` to execute the test and get a comparative diff. To create the test, you should use standard Python first, make that work, then compare.

Reply:
And yes, please revert it, as it actually lowers test coverage.

Reply:
OK, thank you for explanation